Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM golang:1.16
 
 WORKDIR /gotty
 COPY . /gotty
+RUN apt-get update && apt-get install -y nodejs npm
```

Comment:
Move this up before the `COPY` command. That way it won't have to run again every time something in the filesystem changes.